src/utils.js

```diff
@@ -11,4 +11,10 @@ export function parseLength(obj, acceptStr) {
     return new CSSUnitValue(value, unit);
   }
   return null;
+}
+
+const canonicalUnits = new Set(["px", "deg", "s", "hz", "dppx", "number", "fr"]);
```

Comment:
Wondering if we should group these sets of units in one place.  I see canonical units here,  units by type in proxy-cssom and compatible unit types in numeric-value-typing. Not strictly required, but might be easier to spot omissions if in one place.

Reply:
Thanks for pointing this out🙏

I moved most of the units into a common const: https://github.com/flackr/scroll-timeline/blob/ce5f19d9f0c0e012ae53d22fbb3290690aebebc0/src/numeric-values.js#L12

I renamed the file to numeric-values, as it contained both common numeric operations and numeric value typings.

I've tried to keep code for polyfilling the typed OM separate from the rest, as this will only be needed in some browsers. As a consequence there's a small duplication: I have kept the list of canonical units for lookup during simplification.